api/src/operations/request/index.ts

```diff
@@ -16,8 +16,8 @@ export default defineOperationApi<Options>({
 			acc[header] = value;
 			return acc;
 		}, {} as Record<string, string>);
-
-		const result = await axios({ url, method, data: body, headers: customHeaders });
+		
+		const result = await axios({ url: encodeURIComponent(url), method, data: body, headers: customHeaders });
```

Comment:
Just noticed a second thing just in time, i believe using `encodeURI` here is more appropriate as `encodeURIComponent` will encode the full url from `https://directus.io/` to `https%3A%2F%2Fdirectus.io%2F` where you only want to encode the parameters used in the url. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI

Reply:
Yeah! you got a point, Thanks for letting me know